src/components/time-picker/time-picker.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="time-picker" aria-label="Time picker">
 
-    <div class="time-picker-column" [class.has-error]="!_valid" *ngIf="showHours">
+    <div class="time-picker-column time-hours-picker" [class.has-error]="!_valid" *ngIf="showHours">
```

Comment:
Added unique classes for these columns as there was no way of identifying them uniquely.